e2e/specs/unpublish-form.spec.ts

```diff
@@ -16,28 +11,28 @@ test.beforeEach(async ({ api }) => {
   await addFormResources(api, valueReference, form.uuid);
 });
 
-test("Unpublish a form", async ({ page }) => {
+test('Unpublish a form', async ({ page }) => {
   const formBuilderPage = new FormBuilderPage(page);
 
-  await test.step("When I visit the form builder", async () => {
+  await test.step('When I visit the form builder', async () => {
     await formBuilderPage.gotoFormBuilder();
   });
 
-  await test.step("And I search for the form I need to unpublish", async () => {
+  await test.step('And I search for the form I need to unpublish', async () => {
     await formBuilderPage.searchForm(form.name);
   });
 
-  await test.step("And I click on a form I need to unpublish", async () => {
+  await test.step('And I click on a form I need to unpublish', async () => {
     await page.getByRole('row', { name: form.name }).getByLabel('Edit Schema').click();
   });
 
-  await test.step("Then I click on the unpublish form button and confirms the unpublication", async () => {
+  await test.step('Then I click on the unpublish form button and confirms the unpublication', async () => {
     await formBuilderPage.unpublishFormButton().click();
     await formBuilderPage.unpublishFormConfirmationButton().click();
```

Comment:
Watching the video, this doesn't seem to work. There's a failure "Cannot read properties of undefined (reading 'length')". This seems to trigger at the first `unpublishFormButton().click()`.